geopyspark/geotrellis/union.py

```diff
@@ -26,37 +26,16 @@ def union(*layers):
     """
 
     if len(layers) == 1:
-        raise ValueError("Union can only performed on 2 or more layers")
+        raise ValueError("combine_bands can only performed on 2 or more layers")
```

Comment:
Search/replace error ?

Reply:
Yeah, that's definitely a search/replace error.